typography.html

```diff
@@ -0,0 +1,158 @@
+<!doctype html>
```

Comment:
Do we want to include this as part of the commit? A page like this would probably count as being part of the "Pattern Library," and probably shouldn't be kept in the home directory